docs/INSTALLATION.md

```diff
@@ -0,0 +1,265 @@
+* `MYSQL_DATABASE` - The name of the [MariaDB](https://mariadb.org/) database which defaults to `AssetDatabase`.
+* `MYSQL_USER` - The username used to connect to the MariaDB database which defaults to `assetManager`.
+* `MYSQL_PASSWORD` - The username used to connect to the [MariaDB](https://mariadb.org/) database which defaults to `assetManager`.
+* `s3Client__AccessKey` - The access key used to connect to the [S3](https://www.nuget.org/packages/Amazon.S3/) compliant object storage as described in the [Asset Manager documentation](https://github.com/ove/ove-asset-services/blob/master/packages/ove-asset-manager/README.md#configuration).
+* `s3Client__Secret` - The secret used to connect to the [S3](https://www.nuget.org/packages/Amazon.S3/) compliant object storage as described in the [Asset Manager documentation](https://github.com/ove/ove-asset-services/blob/master/packages/ove-asset-manager/README.md#configuration).
+* `s3Client__ServiceURL` - The service URL of the [S3](https://www.nuget.org/packages/Amazon.S3/) compliant object storage as described in the [Asset Manager documentation](https://github.com/ove/ove-asset-services/blob/master/packages/ove-asset-manager/README.md#configuration).
+* `ServiceHostUrl` - Hostname (or IP address) + port of the corresponding Asset Management Service.
+* `MariaDB__ConnectionString` - The connection string used by the [Asset Manager](https://github.com/ove/ove-asset-services/tree/master/packages/ove-asset-manager) to connect to [MariaDB](https://mariadb.org/) database. The format of the connection string must always be similar to what is provided, but the port number, database name, username and password must change accordingly if their default values were changed.
+* `AssetManagerHostUrl` - Hostname (or IP address) + port of the [Asset Manager](https://github.com/ove/ove-asset-services/tree/master/packages/ove-asset-manager).
+
+### Starting and stopping OVE
+
+OVE provides separate installation scripts to help users install the necessary components. To install and start OVE on Docker run:
+
+```sh
+docker-compose -f docker-compose.ove.yml up -d
+```
+
+Replace the `-d` option above with `--no-start` if you want to install OVE but do not want to start it at the end of the installation process.
+
+Once the installation procedure has completed, the successful installation of OVE can be verified by accessing the OVE Core API Documentation (located at: `http://OVE_CORE_HOST:PORT/api-docs`) using a web browser.
+
+To install and start [**OVE Asset Services**](https://github.com/ove/ove-asset-services) on Docker run:
+
+```sh
+docker-compose -f docker-compose.assets.yml up -d
+```
+
+Once the services have started, you can check their status by running:
+
+```sh
+docker ps
+```
+
+The `ps` command will list containers along with their `CONTAINER_ID`. Then, to check logs of an individual container, run:
+
+```sh
+docker logs <CONTAINER_ID>
```

Comment:
We should note this retrieves the pm2 logs

Reply:
Based on how docker logs work, this retrieves all logs. This includes PM2, .NET, and even anything else generated when running the corresponding docker CMD/ENTRYPOINT, or anything written to `stdout` and `stderr` while the container is operational. It will be incorrect to say PM2 logs.